### DISCUSSION TITLE
HA frontend / supervised

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2033] [GENERALE - Discussioni generiche]
Ciao a tutti, 
grazie alle guide sono riuscito ad installare HA tramite docker compose su un Rasperry PI4 con SSD.
prima di terminare le configurazioni però sono andato in inpostazioni/informazioni per vedere quale versione mi aveva insatllato e vedo che mi ha installato la versione Home Assistant 2022.11.1 - Frontend 20221102.1
mi sarei aspettato da guida la supervised ma mi ritrovo la frontend.
sto prendendo un abbaglio o ho sbagliato i link del container?
ringrazio tutti in anticipo!

**Comment by snakuzzo** [comment]
Hai installato core, non la supervised

**Reply by Matty1184** [reply]
E come immaginavo... devo posso trovare i link corretti? 
Cambiando il link nel docker si sovrascrive o meglio crearne un container nuovo?

**Reply by snakuzzo** [reply]
Ma guarda che HA Supervised non si installa così.

Leggi [QUI](https://indomus.it/guide/come-installare-e-configurare-home-assistant-hassio-con-docker-su-un-raspberry-pi-gia-in-uso)

**Reply by Matty1184** [reply]
@snakuzzo mi sta sfuggendo qualcosa! 
leggendo le guide ho capito che su docker posso installare sia Core che Supervised, il consigliato è ovviamente Core ma visto che non sono espertissimo (come si può notare) volevo optare per la Supervised. 
la precedura per il docker ho visto che è identica, ho optato per la versione compose, non riesco però a capire dove si cambia da core a Supervised in quanto entrambe le guide non prevedono la versione compose ma classica

**Reply by snakuzzo** [reply]
Per la Supervised su docker devi usare lo script. Segui la guida.
